cmd/discovery5.go

```diff
@@ -80,6 +83,28 @@ func GenNewPrivKey() (*ecdsa.PrivateKey, error) {
 	return ecdsa.GenerateKey(gcrypto.S256(), rand.Reader)
 }
 
+var EthBootonodes = []*enode.Node{
```

Comment:
Hey @Ryan0v0 , I don't see the point of duplicating the Bootnodes. 
They were already in [pkg/config/bootnodes.go](https://github.com/migalabs/eth-light-crawler/blob/main/pkg/config/bootnodes.go)